[2024-03-29T17:00:55.272+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_scheduling.hello scheduled__2022-03-06T00:00:00+00:00 [queued]>
[2024-03-29T17:00:55.312+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_scheduling.hello scheduled__2022-03-06T00:00:00+00:00 [queued]>
[2024-03-29T17:00:55.316+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-03-29T17:00:55.364+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): hello> on 2022-03-06 00:00:00+00:00
[2024-03-29T17:00:55.379+0000] {standard_task_runner.py:60} INFO - Started process 2975 to run task
[2024-03-29T17:00:55.390+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'demo_scheduling', 'hello', 'scheduled__2022-03-06T00:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/demo_scheduling.py', '--cfg-path', '/tmp/tmppcotm5pg']
[2024-03-29T17:00:55.402+0000] {standard_task_runner.py:88} INFO - Job 30: Subtask hello
[2024-03-29T17:00:55.559+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-29T17:00:55.686+0000] {task_command.py:423} INFO - Running <TaskInstance: demo_scheduling.hello scheduled__2022-03-06T00:00:00+00:00 [running]> on host c4cb237119ff
[2024-03-29T17:00:55.956+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='domwatcharin' AIRFLOW_CTX_DAG_ID='demo_scheduling' AIRFLOW_CTX_TASK_ID='hello' AIRFLOW_CTX_EXECUTION_DATE='2022-03-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-06T00:00:00+00:00'
[2024-03-29T17:00:55.961+0000] {demo_scheduling.py:9} INFO - Hello
[2024-03-29T17:00:55.964+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-29T17:00:56.054+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=demo_scheduling, task_id=hello, execution_date=20220306T000000, start_date=20240329T170055, end_date=20240329T170056
[2024-03-29T17:00:56.143+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-29T17:00:56.228+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
